test/unit/elasticsearch_index_test.rb

```diff
@@ -126,7 +126,7 @@ def test_commit
   end
 
   def test_can_fetch_documents_by_format
-    search_pattern = "http://example.com:9200/mainstream_test/_search?scroll=60m&search_type=scan&size=500"
+    search_pattern = "http://example.com:9200/mainstream_test/_search?scroll=1m&search_type=scan&size=500"
```

Comment:
I would generally prefer not including the test domain if possibel.

I think the below would work (needs to be tested)

```
search_pattern = %r{/mainstream_test/_search?scroll=1m&search_type=scan&size=500$}
```